string name

---

uint64 code

string message
